aesara/tensor/random/op.py

```diff
@@ -26,8 +26,11 @@
 
 
 def default_shape_from_params(
-    ndim_supp, dist_params, rep_param_idx=0, param_shapes=None
-):
+    ndim_supp: int,
+    dist_params: Sequence[Variable],
```

Comment:
It looks like you might need to use the `Sequence` type from `typing` instead of `collections.abc`.

Reply:
Hi yes, you are right. 

So we can have two options: 
1. Use `typing.Sequence`, and keep the `Sequence` for `abc` 
2. Use `abc.Sequence`, and `Sequence` acts as `typing.Sequence`.

I will prefer option 2, since we only need to change `Sequence` to `abc.Sequence` once at line [L144](https://github.com/aesara-devs/aesara/blob/4e2fc615b32df85ff16f17f2595d7f74e659b034/aesara/tensor/random/op.py#L144). 

But let me know if you prefer the other option.

Thank you :D

Reply:
We should only need to change the import from `collections.abc` to `typing`.

Reply:
So when we remove `collection.abc`, this code stills work rights?

`if not isinstance(self.ndims_params, Sequence)` # I thought the `Sequence` in this line need to be from `collection.abc`.

Reply:
Yes, `typing.Sequence` should work in that case.